schemas/context/implementation.schema.json

```diff
@@ -0,0 +1,35 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/implementation",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Implementation",
+  "type": "object",
+  "meta:extensible": true,
+  "description": "Implementation of an API.",
+  "definitions": {
+    "implementation": {
+      "properties": {
+        "xdm:name": {
+          "title": "Name",
+          "type": "string",
```

Comment:
If there is a reasonable expectation that this value will be filtered or otherwise processed, it can make sense to use a URI to make the value truly unique. Otherwise `sCode` and `Analytics S-Code` will be treated as different values.

Reply:
Yes, makes sense to state it is a URI